src/components/AddItem/AddItem.jsx

```diff
@@ -1,3 +1,168 @@
+import styles from "./AddItem.module.scss";
+import { useState, useRef } from "react";
+
 export default function AddItem() {
-  return <div>addItem</div>;
+  const [imagePreview, setimagePreview] = useState(null);
+  const [formData, setFormData] = useState({
+    image: "",
+    name: "",
+    about: "",
+    price: "",
+    tag: [],
+  });
+  const inputRef = useRef();
+  const active = Object.values(formData).every((value) =>
```

Comment:
💊 제안
boolean 타입임을 알 수 있고 어떤 값인지 알 수 있는 이름을 추천드려요~

```suggestion
  const isFormValid = Object.values(formData).every((value) =>
```

Reply:
❗️ 수정요청
요구사항에서 이미지가 없어도 '등록' 버튼이 활성화되게 하라고 되어 있어요~ 확인 후 요구사항대로 구현하시면 더 좋을 것 같아요!